packages/commonwealth/client/scripts/views/components/feed.tsx

```diff
@@ -19,23 +19,28 @@ const DEFAULT_COUNT = 10;
 
 export const Feed = (props: FeedProps) => {
   const [loading, setLoading] = useState<boolean>(true);
-  const [loadingMore, setLoadingMore] = useState<boolean>(false);
```

Comment:
FFR since we're passing true/false as initial value we shouldn't need to pass the type, as it can be inferred.